src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -71,7 +71,7 @@ public TagType getTagType() {
         return tagType;
     }
 
-    private static TagType getTagType(TagStatus ts) {
+    public static TagType getTagType(TagStatus ts) {
```

Comment:
Consider renaming this or the method below.
They have different purposes but are sharing the same name.